pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -637,6 +637,18 @@ self: super: builtins.intersectAttrs super {
   cairo = addBuildTool super.cairo self.buildHaskellPackages.gtk2hs-buildtools;
   pango = disableHardening (addBuildTool super.pango self.buildHaskellPackages.gtk2hs-buildtools) ["fortify"];
 
+  # https://github.com/dhall-lang/dhall-haskell/pull/1422
```

Comment:
Could you expand this comment so that the nixpkgs maintainers will have a better idea when these overrides can be removed (without having to follow the link)?

Reply:
Done. Link was wrong.
https://github.com/dhall-lang/dhall-haskell/commit/dedd5e0ea6fd12f87d887af3d2220eebc61ee8af